identifiers/country-se/constituencies.csv

```diff
@@ -0,0 +1,30 @@
+id,name
+ocd-division/country:se/ed:1,Municipality of Stockholm

```

Comment:
I know in the Elections Act these are listed by a numbered list, but are the numbers meaningful to the identity of the districts? If not, we should use the names of the constituencies for the identifiers instead.

Reply:
I don't think the numbers are meaningful since they aren't used in the Official website.

Reply:
In that case, I think the names of the constituencies should be part of the OCD IDs so that we don't end up with historical data being interpreted wrong after a redistricting happens.

Reply:
Makes sense. I updated the ids. Please let me know if I need more details to the id (example: _county)